File: class_object.cpp
#include <bits/stdc++.h>
using namespace std;

class Car{
    public:
    Car(){
        cout << "constructor method" << endl;
    }
    void carName(string carName){
       cout << carName << endl;
    }
    string display(string carColor){
        return carColor;
    }
    ~Car(){
        cout << "destrcutor method" << endl;
    }
};

int main(){
    Car c1;
    c1.carName("BMW");
    c1.carName("BOLVO");
    cout << c1.display("Red") << endl;
    return 0;
};

File: class_object.py
class Car:
    def __init__(self) -> None:
        print("Hello")
    
    def car_name(carname):
        print(carname)
    
    def __del__(self):
        print("Goodbye")
    def __str__(self) -> str:
        return "a car object"



car1 = Car()
Car.car_name("Bolvo")
Car.car_name("BMW")
print(car1)